en/CONTRIBUTING.md

```diff
@@ -1,3 +1,50 @@
 # Contribution Guidelines
 
-Please note that this project is released with a [Contributor Code of Conduct](../CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
+:sparkles::heart: Thank you so much for taking the time to contribute! :heart::sparkles:
+
+The following is a set of guidelines for contributing to any of our projects, which are hosted in the [FolksWhoCode Organization](https://github.com/folkswhocode) on GitHub. Feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[Code of Conduct and Values](#code-of-conduct-and-values)
+
+[How Can I Contribute?](#how-can-i-contribute)
```

Comment:
We have to fix these list elements: first and fourth elements are not indented and they don't have the `*` character to make them a markdown list element 😄